src/inventory.cpp

```diff
@@ -576,6 +577,12 @@ void inventory::form_from_map( map &m, std::vector<tripoint> pts, const Characte
             continue;
         }
         vehicle *const veh = &vp->vehicle();
+        if( auto ovi = checked_vehi.find( veh ); ovi == checked_vehi.end() ) {
```

Comment:
```suggestion
        if( auto ovi = checked_vehi.find( veh ); ovi == checked_vehi.end() ) {
```
let's use https://en.cppreference.com/w/cpp/container/unordered_map/contains

Reply:
Those suggestions don't seem to differ at all?

Reply:
fixed